.clang-tidy

```diff
@@ -29,19 +39,24 @@ hicpp-avoid-goto,
 misc-unused-alias-decls,
 misc-unused-using-decls,
 modernize-*,
+-modernize-avoid-c-arrays,
 -modernize-concat-nested-namespaces,
+-modernize-macro-to-enum,
 -modernize-return-braced-init-list,
 -modernize-use-auto,
 -modernize-use-default-member-init,
 -modernize-use-using,
 -modernize-use-trailing-return-type,
 -modernize-use-nodiscard,
+-modernize-redundant-void-arg,
 performance-*,
+-performance-move-const-arg,
 -performance-noexcept-move-constructor,
 -performance-unnecessary-value-param,
+-performance-no-int-to-ptr,
 '
 HeaderFilterRegex: '^(c10/(?!test)|torch/csrc/(?!deploy/interpreter/cpython)).*$'
 AnalyzeTemporaryDtors: false
 WarningsAsErrors: '*'
-CheckOptions:
+CheckOptions: []
 ...
```

Comment:
I actually don't know how this ellision works, I assumed it was being replaced with it's on set of check options args.

Reply:
My clang-tidy-16 complaints that consistently, so I have to provide a value for CheckOptions